StreetTagged/AppDelegate.swift

```diff
@@ -273,5 +309,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UITabBarControllerDelegat
             }
         }
     }
+    
+    func getSettingsQuery() {
+        appSyncClient?.fetch(query: GetQuery()) {
+            (result, error) in
+            if error != nil {
+                print(error?.localizedDescription ?? "")
+                return
+            }
+            
+            
+            firstName = (result?.data?.get?.firstName)!
+            lastName = (result?.data?.get?.lastName)!
+            bio = (result?.data?.get?.bio)!
+            location = (result?.data?.get?.location)!
+            username = (result?.data?.get?.username)!
+            avatarImage = (result?.data?.get?.image)!
+            
+        }
+        
+    }
+    
+    func saveProfile(firstName:String,lastName:String, bio:String, image:String,location:String) {
```

Comment:
@johnosullivan here is the function you need to run to save your user profile data to dynamo